orchagent/main.cpp

```diff
@@ -149,6 +149,20 @@ int main(int argc, char **argv)
                 exit(EXIT_FAILURE);
             }
             break;
+        case 't':
+        {
+            gMacAgingTimeSet = true;
+            int tmptime = stoi(optarg);
+            if (tmptime < 0)
+            {   
+                macAgingTime = 0;
+                SWSS_LOG_DEBUG("Input macAgingTime < 0, set macAgingTime to 0");
```

Comment:
if we don't provide the parameter, what's the default value. do we need to explicitly set the value?